src/data/lovelace_custom_cards.ts

```diff
@@ -0,0 +1,23 @@
+export interface CustomCardEntry {
+  type: string;
+  name?: string;
+  description?: string;
+  preview?: boolean;
+}
+
+export interface CustomCardsWindow {
+  customCards?: CustomCardEntry[];
+}
+
+export const getCustomCards = () => {
+  return ((window as CustomCardsWindow).customCards || []).filter((card) => {
+    return card.type !== undefined;
+  });
+};
+
+export const getCustomCardEntry = (type: string) => {
+  if (type.startsWith("custom:")) {
```

Comment:
I don't understand why we would require `type` to be passed in with `custom:` as a prefix?

Reply:
The `custom:` prefix is added on all the way from `render()` and is necessary to get the correct stub config if the card doesn't specify one.
As it stands now, customCard takes precedence, so without this check, an entry in `window.customCards` could override the description and name of stock cards.

Reply:
Then again, it is force added to the filter on the line right below this one... 🤦‍♂